src/conn/conn_layered.c

```diff
@@ -1091,6 +1091,47 @@ __layered_move_updates(
     return (ret);
 }
 
+/*
+ * __layered_clear_ingest_table --
+ *     After ingest content has been drained to the stable table, clear out the ingest table.
+ */
+static int
+__layered_clear_ingest_table(WT_SESSION_IMPL *session, const char *uri)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *internal_session;
+
+    WT_ASSERT(session, WT_SUFFIX_MATCH(uri, ".wt_ingest"));
+
+    internal_session = NULL;
+
+    WT_RET(
```

Comment:
We already opened a internal session in __layered_drain_ingest_tables. I don't think you need to open another internal session.

Reply:
Sure, but I don't like making this depend on how another function calls it.

Reply:
We already passing in a session here.